packages/tanstack-react-query/src/internals/createOptionsProxy.ts

```diff
@@ -1,4 +1,9 @@
-import type { DataTag, QueryClient, QueryFilters } from '@tanstack/react-query';
+import {
+  useQuery,
```

Comment:
This might be rejected by Next.js if using server components like this: https://trpc.io/docs/client/tanstack-react-query/server-components